pkg/kubecost/asset.go

```diff
@@ -3163,6 +3179,20 @@ func (as *AssetSet) accumulate(that *AssetSet) (*AssetSet, error) {
 	return acc, nil
 }
 
+func (as *AssetSet) MonitoredPropsOfCloudCostItem(cci *CloudCostItem) (time.Time, string, string) {
+	for _, node := range as.Nodes {
+		if node.MonitoringKey() == cci.MonitoringKey() {
+			props := node.GetProperties()
+			if props == nil {
+				continue
+			}
+			// To-DO: No match between cluster ID and Node here?
+			return node.End, props.Cluster, ""
```

Comment:
@nikovacevic , i want your opinion here, i need to return clusterID and asset properties don't have clusterID only clusterName. Is there anything i missed here?

Reply:
fixed downstream so resolving the conversation!